driver/pace/driver/report.py

```diff
@@ -0,0 +1,146 @@
+import copy
+import dataclasses
+import json
+from datetime import datetime
+from typing import Any, Dict, List
+
+import numpy as np
+
+from pace.util.mpi import MPI
+
+
+@dataclasses.dataclass
+class Experiment:
+    dataset: str
+    format_version: int
+    git_hash: str
+    timestamp: str
+    timesteps: int
+    version: str
+
+
+@dataclasses.dataclass
+class TimeReport:
+    hits: int
+    times: list
+
+
+@dataclasses.dataclass
+class Report:
+    setup: Experiment
+    times: dict
+    SYPD: float
```

Comment:
would it not be more interesting to store `dt_atmos` and `__post_init__` calculate the SYPD?

Reply:
That makes sense, changed to calculate SYPD in post init.